project.clj

```diff
@@ -4,7 +4,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [clojail "1.0.6"]]
+                 [clojail "1.0.6"]
+                 [org.clojure/core.match "0.3.0-alpha5"]]
```

Comment:
I'd prefer it if you didn't pull in `core.match` for a simple feature like this.